tldr/api.py

```
@@ -451,24 +451,31 @@ def _get_module_exports(
+            break
 
-    if not module_file.exists():
+    if module_file is None:
         # Try as directory with __init__.py (Python package)
         init_file = project / module_path / "__init__.py"
         if init_file.exists():
             module_file = init_file
         else:
-            raise ValueError(f"Module not found: {module_path} (tried {module_file} and {init_file})")
+            tried = ", ".join(str(project / f"{module_path}{e}") for e in extensions)
+            raise ValueError(f"Module not found: {module_path} (tried {tried} and {init_file})")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)